Cargo.toml

```diff
@@ -9,3 +9,6 @@ members = [
     "ndk-sys",
     "cargo-apk",
 ]
+
+[patch.crates-io]
+cargo-subcommand = { git = "https://github.com/dvc94ch/cargo-subcommand", branch = "master" }
```

Comment:
Will need to make a new `cargo-subcommand` release before we get this in - but give it some time to finalize this first.